### ISSUE TITLE
Manifest columns in different order than table (simple mode)

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/db-extractor-pgsql#32]
ref https://keboola.zendesk.com/agent/tickets/9291
job https://connection.keboola.com/admin/projects/976/jobs/452290290
config https://connection.keboola.com/admin/projects/976/extractors/keboola.ex-db-pgsql/452290048

create table event

![image](https://user-images.githubusercontent.com/497675/46869887-66e60180-ce2d-11e8-9b4e-f6c0302de56a.png)
https://connection.keboola.com/admin/projects/976/jobs/452290290?q=&eventId=1061338814

import event

![image](https://user-images.githubusercontent.com/497675/46869858-559cf500-ce2d-11e8-9745-a0efd284302b.png)
https://connection.keboola.com/admin/projects/976/jobs/452290290?q=&eventId=1061338964

csv content
![image](https://user-images.githubusercontent.com/497675/46869895-6ea5a600-ce2d-11e8-9ebf-e4efc4e3a80d.png)

manifest
```
    "columns": [
        "vip_tier",
        "***_user",
        "objectid",
        "created_at",
        "***_item_common"
    ]
```

maybe this?

![image](https://user-images.githubusercontent.com/497675/46869971-b0cee780-ce2d-11e8-8925-9856140b49e7.png)

**Comment by ondrejhlavacek** [comment]
I was not able to reproduce it on a testing db.

**Comment by pivnicek** [comment]
The only thing that comes to mind here is that the column order retrieved when executing `select * from table` is different than the orderdinal values of the columns as listed in the information schema.

I will try to confirm / reproduce

**Comment by pivnicek** [comment]
fixed in 2.3.1
